src/stories/ChatView.stories.tsx

```diff
@@ -21,27 +23,35 @@ const errorStore = new TextStreamStore();
 
 const withProviders =
   (
+    messages: ChatMessage[],
```

Comment:
Only big adjustment had to come for storybook.

Each story can still set its own messages, just not via props, but via `messageHistoryStore.setMessages(...)`